installer/bundle/bundle_skel_AIX.sh

```diff
@@ -343,16 +439,16 @@ case "$installMode" in
         echo "Updating cross-platform agent ..."
         check_if_pkg_is_installed omi
         if [ $? -eq 0 ]; then
-            pkg_upd $OMI_PKG omi
-            # It is acceptable that this fails due to the new omi being
-            # the same version (or less) than the one currently installed.
-            OMI_EXIT_STATUS=0
+            shouldInstall_omi
+            pkg_upd $OMI_PKG omi $?
```

Comment:
I actually remember this. I specifically worked out shouldInstall_omi to "do the right thing" if the package was not installed. See line 226. I could have also called check_if_pkg_is_installed, but it wasn't necessary to do that AND check the version, I could essentially do both in one shot.

This is how I did it on Linux, and without a compelling reason to deviate, I think the could should be as identical as possible, with a goal of making it 100% identical (and I think it is right now). This just eases maintenance down the road.